### DISCUSSION TITLE
Student Developer pack claim

### DISCUSSION THREAD

**Discussion by dsp0205** [github/india#548] [Students]
I have registered for the Future Ready talent Microsoft virtual internship program in the first batch. 
In order to claim GitHub Student dev pack, I have updated Microsoft Azure as my skill in my LinkedIn profile a long time ago. Still I am not able to access student dev pack and the status shows the message that "your request is submitted", for a long time now.
Can anyone guide me through this?
![2021-11-18 (3)](https://user-images.githubusercontent.com/55679683/142420883-db384f98-fdd3-4a99-b103-4025f630b052.png)

**Comment by divyaaaaaa12** [comment]
Facing the same problem!

**Reply by Ayan-thecodeking** [reply]
May be the proof you have submitted is not acceptable

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official GitHub Externship Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Link to join: https://discord.gg/ceMXzhfaka
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏
